hw/top_earlgrey/earlgrey_xilinx_prim_pkg.core

```diff
@@ -0,0 +1,19 @@
+CAPI=2:
+# Copyright lowRISC contributors (OpenTitan project).
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+name: "lowrisc:prim_xilinx:earlgrey_xilinx_prim_pkg"
+description: "Single port RAM"
+virtual:
+  - lowrisc:prim_xilinx:prim_xilinx_pkg
```

Comment:
This requires FuseSoC v2.4 to work properly, no?

Reply:
No, we use virtual cores with my fusesoc fix already. I added the satsolver instructions to our fork at the same time as upstream.

If we use upstream fusesoc, _then_ 2.4 is the first release that includes my commits.